tools/go-ls-tags/cli/logger.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2022 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cli
+
+import (
+	"context"
+	"os"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+	"knative.dev/test-infra/pkg/logging"
+)
+
+func contextWithLogger() context.Context {
+	ctx := context.Background()
+	fallback := logging.FromContext(ctx)
+	return logging.WithLogger(context.Background(), newLogger(fallback))
```

Comment:
Nit: You can either save some line by 
```
fallback := logging.FromContext(context.Background())
```
or better use the `ctx` one line below as well:
```
return logging.WithLogger(ctx, newLogger(fallback))
```

Reply:
Yes, this is weird. The `context.Background()` should be executed only once...

Reply:
Fixed in c71881c5f700a78b006f2a5835b6008222bb6a87